CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 ## v71.0.0-SNAPSHOT - unreleased
 
-
-### ⚙️ Technical
-* Misc. Improvements to ViewManager
+### 💥 Breaking Changes (upgrade difficulty: 🟢 LOW - changes to new ViewManager APIs)
```

Comment:
I think we could take a liberty here and not book this as a breaking change, especially if we have no other reason to jump to a new major version.